lib/DDG/Spice/NewYorkTimes.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::NewYorkTimes;
+# ABSTRACT: Details on New York Times best sellers
+
+use strict;
+use DDG::Spice;
+
+spice to => 'https://api.nytimes.com/svc/books/v3/lists/best-sellers/history.json?api-key={{ENV{DDG_SPICE_NewYorkTimes_APIKEY}}}&title=$1';
```

Comment:
API KEY Variable should be all caps: `DDG_SPICE_NEWYORKTIMES_APIKEY`

We'll need to register for an API Key before we an get this on Beta

/cc @tagawa